sigma/pipelines/panther/sentinelone_panther_pipeline.py

```diff
@@ -0,0 +1,304 @@
+from sigma.processing.conditions import LogsourceCondition, RuleProcessingItemAppliedCondition
+from sigma.processing.pipeline import ProcessingItem, ProcessingPipeline, QueryPostprocessingItem
+from sigma.processing.transformations import (
+    AddConditionTransformation,
+    ChangeLogsourceTransformation,
+    FieldMappingTransformation,
+    RuleFailureTransformation,
+)
+
+from sigma.pipelines.panther.sdyaml_transformation import SdYamlTransformation
+
+
+def sentinelone_panther_pipeline() -> ProcessingPipeline:
+    translation_dict = {
+        "process_creation": {
+            "ProcessId": "tgt_process_pid",
+            "Image": "tgt_process_image_path",
+            "Description": "tgt_process_displayName",  # Not sure whether this should be Description or Product???
```

Comment:
Does this have to be one of those two?

Reply:
To be honest, I'm not sure, but these are logical options for the field `Description`

Reply:
Why not `ProcessName` then?

Reply:
This comment is from [here](https://github.com/7RedViolin/pySigma-backend-sentinelone/blob/master/sigma/pipelines/sentinelone/sentinelone.py#L41), I thought it was yours 😅 

Let's leave the same mapping on our end.